lib/_http_outgoing.js

```diff
@@ -382,7 +382,7 @@ OutgoingMessage.prototype.setHeader = function setHeader(name, value) {
 
 
 OutgoingMessage.prototype.getHeader = function getHeader(name) {
-  if (arguments.length < 1) {
+  if (typeof name !== 'string') {
```

Comment:
Good point... hmmm... yeah, there may not be a way around making this a semver-major